docs/content/advanced/backend/subscribers/create-subscriber.md

```diff
@@ -0,0 +1,92 @@
+---
+
+title: Create a Subscriber
+
+---
+
+# Create a Subscriber
+
+In this document, you’ll learn how you create a subscriber in your Medusa server that listens to events to perform an action.
+
+## Overview
+
+In Medusa, there are events that are emitted when a certain action occurs. For example, if a customer places an order, the `order.placed` event is emitted with the order data. 
+
+The purpose of these events is to allow other parts of the platform, or third-party integrations, to listen to those events and perform a certain action. That is done by creating subscribers.
+
+Subscribers register handlers for an events and allows you to perform an action when that event occurs. For example, if you want to send your customer an email when they place an order, then you can listen to the `order.placed` event and send the email when the event is emitted.
+
+Natively in Medusa there are subscribers to handle different events. However, you can also create your own custom subscribers. 
+
+Custom subscribers reside in `src/subscribers` within a JavaScript file. Each file holds a class, which is the subscriber. By convention, the class name should end with `Subscriber` and the file name should be the camel-case version of the class name without `Subscriber`. For example, the `WelcomeSubscriber` class is in the file `src/subscribers/welcome.js`.
+
+Whenever an event is emitted, the subscriber’s registered handler method is executed. The handler method receives as a parameter an object that holds data related to the event. For example, if an order is placed the `order.placed` event will be emitted and all the handlers will receive the order id in the parameter object.
+
+## Prerequisites
+
+Before you can use subscribers, whether your custom subscribers or Medusa’s subscribers, you need to have [Redis](https://redis.io) installed and setup on your machine.
```

Comment:
Medusa's event system works by pushing data to a Queue that each handler then gets notified of. The queuing system is based on Redis and you will therefore need to make sure that Redis is installed and configured for your Medusa project.

Reply:
Let's try to give the context to why the developer needs to install additional software.